src/contact.html

```diff
@@ -28,8 +29,37 @@
         </nav>
     </header>
     <main class="content">
-        <!-- TODO -->
+        <div class="contact_text">
+            <p>
+                <b>Country:</b>
+                <br>
+                <b>United States of America</b>
+                <br>
+                <br>
+                <b>Staples Center:</b>
+                <br>
+                <b>1111 S. Figueroa St.</b>
+                <br>
+                <b>Los Angeles, CA 90015</b>
+                <br>
+                <br>
+                <b>UCLA Health Training Center:</b>
+                <br>
+                <b>2275 E. Mariposa Ave.</b>
+                <br>
+                <b>El Segundo, CA 90245</b>
+                <br>
+                <br>
+                <a class="contact_mail" href="mailto:Max.Mustermann@hotmail.com">Max Mustermann, Max.Mustermann@hotmail.com</a>
```

Comment:
@MiljkovicD Please make sure, that the `mailto:link` opens in a new browser tab to avoid leaving the current page on your website.